eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -37,13 +37,11 @@
           }
       },
       "TestMode": {
-          "netcoreapp21_Coverage_Record": {
```

Comment:
These changes look unrelated to the issue you are fixing. Is there a reason you decided to remove the net5.0 configuration?

Reply:
It is somewhat related. This configuration wasn't passing any value of `AdditionalTestArguments` but we already have net5.0 in other generated configurations. So instead of fixing it I've removed it.